fe/src/components/game/PlayerHistory.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useContext } from "react";
+import styled from "styled-components";
+import { GlobalContext } from "../../App";
+
+const PlayerHistory = ({ history }) => {
+  const { currHitter } = useContext(GlobalContext);
+
+  // const currHitterInfo = localStorage.getItem("currHitter");
+  // const currHitter = currHitterInfo.currHitter;
+  // console.log(JSON.parse(currHitterLocal));
+
+  const actionType = {
+    S: "스트라이크",
+    B: "볼",
+    H: "안타",
+  };
+
+  const lastHistory = history.historyList[history.historyList.length - 1];
```

Comment:
```suggestion
  const lastHistory = history.list[history.list.length - 1];
```
불필요한 context 추가하지 않는것이 좋긴 합니다ㅎㅎ
https://github.com/ryanmcdermott/clean-code-javascript#dont-add-unneeded-context